docs/estimators/index.rst

```diff
@@ -11,12 +11,76 @@ estimators - High level estimators
 Introduction
 ============
 The `gammapy.estimators` submodule contains algorithms and classes
-for high level flux and significance estimation such as TS maps,
-flux points and light curves.
+for high level flux and significance estimation such as flux maps,
+flux points, flux profiles and flux light curves. All estimators
+feature a common API and allow to estimate fluxes in bands of reconstructed
+energy.
+
+The core of any estimator algorithm is hypothesis testing: a reference
+model or counts excess is tested against a null hypothesis. From the
+best fit reference model a flux is derived and a corresponding sqrt(TS)
+value from the difference in fit statistics to the null hypothesis,
+assuming one degree of freedom (TODO: link to stats). In this case
+sqrt(TS) represents an approximation of the classical significance.
```

Comment:
We need a proper discussion of significance vs sqrt(TS) somewhere.

Reply:
For now I added a reference to `gammapy.stats`. I think this should be the single place where we explain thi sin more detail...

Reply:
In the stats section I added the "recipe" how derive the significance from delta_ts in general. It's not yet in a clear state, but at least the information is somewhere...we can still further improve this of course.